signingscript/tests/test_sign.py

```diff
@@ -6,6 +6,7 @@
 import re
```

Comment:
Backported @tomprince's solution to fix this from https://github.com/mozilla-releng/scriptworker-scripts/pull/125/ in order to unblock the l10n bumper fixing.